src/algorithms/post/election_post.md

```diff
@@ -196,4 +202,10 @@ Each reported fault carries a penality with it.
 
 Note that surprise PoSt will frequently be used by small miners who do not win blocks, and by miners as they are onboarding power to the network, since they will not be able to win any blocks (they have no power) and hence gain power through ElectionPoSt.
 
-While it resets a miner’s ElectionWindow, miners earn no reward from submitting SurprisePoSt messages. This mechanism does forces a miner to announce their faults and clean up the Power Table accordingly.
+While it resets a miner’s ElectionWindow, miners earn no reward from submitting SurprisePoSt messages. This mechanism does force a miner to announce their faults and clean up the Power Table accordingly.
+
+# Miner Onboarding
+
+Storage Power Consensus participants are subject to a {{<sref min_miner_size>}}, meaning miners smaller than `MIN_MINER_SIZE_STOR` of active (or in-deal) storage or whose active storage represents less than `MIN_MINER_SIZE_PERC`% of the network's cannot produce valid electionPoSts. 
```

Comment:
This parenthetical is the only definition of active power in the spec. Please elevate it somewhere appropriate, along with a description of what the difference is from total power, inactive power etc.

Reply:
Should be `and`? 
However, it does not make sense to me to have the 2nd condition. If we asked one miner to have more than 1/3 power to start mining, the first one might be able to control the network. We may need to have a start point for all miners to prevent it.

Reply:
In-deal or not does not determine whether power is active or inactive. Power has to come from in-deal data. Active Power comes from data in Active sectors (and hence the deals are also active). Inactive Power comes from data in Committed, Recovering and Failing sectors. Miners still need to maintain pledge collateral for Inactive Power.

Reply:
Thanks for clarifying that @zixuanzh. i was imprecise (read plain wrong) on my distinction. I will fix accordingly

Reply:
To your point that this needs a clear single def @anorth. Will add it.

@steven004 no I stand by `or`. This is more about liveness than control @steven004. If your powerful miner is the only miner, then yes they control the network. But if any other miners have `>MIN_MINER_SIZE_PERC` or `>MIN_MINER_SIZE_STOR` they can also publish blocks, so control is dependent on power, as it should be.

The second condition is mostly here to ensure that if the network drops to a size `< MIN_MINER_SIZE_STOR/MIN_MINER_SIZE_PERC`, the chain can still grow (otherwise you lose liveness entirely).